panel/io/cache.py

```diff
@@ -427,7 +465,7 @@ async def wrapped_func(*args, **kwargs):
             return ret
     else:
         @functools.wraps(func)
-        def wrapped_func(*args, **kwargs):
+        def wrapped_func(*args: P.args, **kwargs: P.kwargs) -> R:
```

Comment:
I think you may have inadvertently included this file?

Reply:
Thx. I believe I've "undone" it by running `git checkout main -- panel/io/cache.py`.